package OOP.Tests.PartC.TestInvokeMultipleSucc;

import OOP.Provided.OOPMultipleException;

public class C1 implements I1 {
    public Integer foo(Integer a) throws OOPMultipleException {
        return a;
    }
}
